.github/workflows/release.yml

```diff
@@ -2,8 +2,7 @@ name: Release
 on:
   push:
     tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
-      - '[0-9]+.[0-9]+.[0-9]+-rc[0-9]+'
+      - '^[0-9]+.[0-9]+.[0-9]+$'
```

Comment:
as per slack discussion:
```suggestion
      - '[0-9]+.[0-9]+.[0-9]+'
      - '![0-9]+.[0-9]+.[0-9]+-rc[0-9]+'
```

Reply:
Ah, sorry about that. I didn't see your comment here and just put this in directly.